dev_docs/HOW_TO.md

```diff
@@ -161,3 +161,25 @@ be used for sending and receiving protobuf models internally, they are not compa
 by Elasticsearch.
 If you need to encode to JSON an APM event to send it to Elasticsearch, you need to use `MarshalFastJSON`: this method
 will map the protobuf model to an internal format compatible with Elasticsearch expected document and encode it to JSON.
+
+## How to release new `apm-data` version
+
+Once you have merged your PR into the main branch, you will need add a tag so that you can use it for your subsequent PR in `apm-server` or even [MIS](https://docs.elastic.dev/mis).
```

Comment:
```suggestion
Once you have merged your PR into the main branch, you will need add a tag so that you can use it for your subsequent PR in `apm-server`.
```
apm-data is public.